homeassistant/helpers/event.py

```diff
@@ -1,18 +1,27 @@
 """Helpers for listening to events."""
-from datetime import timedelta
 import functools as ft
-from typing import Callable
+import logging
+from datetime import timedelta
+from typing import Any, Callable, Mapping, Optional, Union
 
 import attr
-
-from homeassistant.loader import bind_hass
-from homeassistant.helpers.sun import get_astral_event_next
-from homeassistant.core import HomeAssistant, callback
-from homeassistant.const import (
-    ATTR_NOW, EVENT_STATE_CHANGED, EVENT_TIME_CHANGED, MATCH_ALL,
-    SUN_EVENT_SUNRISE, SUN_EVENT_SUNSET, EVENT_CORE_CONFIG_UPDATE)
-from homeassistant.util import dt as dt_util
-from homeassistant.util.async_ import run_callback_threadsafe
+import voluptuous as vol
+
+from ..const import (
```

Comment:
I'm not a fan of these relative imports. Why change it?

Reply:
Can change it back no probs. Probably just did a sort imports on it.

Reply:
Let's change it back.